src/string.cr

```diff
@@ -2419,7 +2419,7 @@ class String
   # "abcdef".compare("ABCDEF", case_insensitive: true) # => 0
   # "abcdef".compare("ABCDEG", case_insensitive: true) # => -1
   #
-  # "heIIo".compare("heııo", case_insensitive: true, Unicode::CaseOptions::Turkic) # => 0
+  # "heIIo".compare("heııo", case_insensitive: true, options: Unicode::CaseOptions::Turkic) # => 0
```

Comment:
```suggestion
  # "heIIo".compare("heııo", case_insensitive: true, options: :turkic) # => 0
```

Let's keep it short :)

Reply:
Hm actually for Flags enums it might be nicer to keep the full name.
What if you decided to add another option? You can't just write `:option1 | :option2` ... hm or can you? I haven't tried